sdk/maps/maps-route-rest/review/maps-route.api.md

```diff
@@ -4,15 +4,16 @@
 
 ```ts
 
+import { AbortSignalLike } from '@azure/abort-controller';
 import { AzureKeyCredential } from '@azure/core-auth';
```

Comment:
@Lakicar95 @dubiety Not sure if you have discussed this before. I found that there are lot of breaking changes in this PR, especially for the lro. Though Route is in beta, we may still want to consider this carefully.
If you already discussed this, please ignore this message. Just make sure we inform the breaking changes in the CHANGELOG.md

Reply:
@andykao1213 It is changed a little bit how poller is used - which is the consequences of using "latest" tag for @autorest/typescript in the Swagger.md file. What is your suggestion to add to CHANGELOG.md file?

Reply:
Sounds good to me. Thanks!